contracts/BaseStrategy.sol

```diff
@@ -734,119 +435,47 @@ abstract contract BaseStrategy {
             }
             debtPayment = debtOutstanding.sub(loss);
         } else {
-            // Free up returns for Vault to pull
             (profit, loss, debtPayment) = prepareReturn(debtOutstanding);
         }
+        if (enforeChangeLimit) {
+            uint256 totalDebt = vault.strategies(address(this)).totalDebt;
+            require(profit <= (totalDebt * changeLimitRatio) / MAX_BPS);
+            require(loss <= (totalDebt * changeLimitRatio) / MAX_BPS);
```

Comment:
not sure if we need different values for each ratio to check, loss ratio may be 0% while profit ratio can be 1%

Reply:
I think we should but would like other strategists to give their opinions. 
@retro-wavey

Reply:
yes please. default should be 0 for loss in my opinion. and strategist can raise it if it is an unusual strategy that has regular losses